Formula/pypy3@3.8.rb

```diff
@@ -0,0 +1,222 @@
+class Pypy3AT38 < Formula
+  desc "Implementation of Python 3 in Python"
```

Comment:
I've updated the description @henryiii.

Since you're already reviewing this PR, how do you feel about this proposal:
* Rename the current `pypy` package to `pypy@2.7`
* Rename the  `pypy3` package to to `pypy@3.7`
* Rename this package to `pypy@3.8`

Reply:
I agree with @carlocab, the @ refers to versions, rather than flavors/dialects. `pypy38` or `pypy3.8` both sound fine. `pypy3` could be an alias to `pypy3.7`. If there's a pypy 7.4, we could possibly want a `pypy3.8@7.3` for a while, for example.